modules/custom-status/custom-status.php

```diff
@@ -321,7 +315,30 @@ function action_admin_enqueue_scripts() {
 			));
 		}
 
+	}
+
+	function load_scripts_for_block_editor(){
+		// ToDo: This is duplicated in 3 places, should be refactored
```

Comment:
Maybe we can move this check into the `init()` method above? Then it wouldn't be duplicated so much. Something like:

```
if ( ! $this->disable_custom_statuses_for_post_type() ) {
	// Load CSS and JS resources that we probably need in the admin page
	add_action( 'admin_enqueue_scripts', array( $this, 'action_admin_enqueue_scripts' ) );

	// Assets for block editor UI.
	add_action( 'enqueue_block_editor_assets', array( $this, 'load_scripts_for_block_editor') );

	// Assets for iframed block editor and editor UI.
	add_action( 'enqueue_block_editor_assets', array( $this, 'load_styles_for_block_editor') );
}
```

Reply:
Good call, makes sense to me